islandora_scholar.module

```diff
@@ -219,7 +219,7 @@ function islandora_scholar_romeo_access($object) {
  */
 function islandora_scholar_citation_access($object) {
   return variable_get('islandora_scholar_users_choose_display_csl', FALSE) &&
-    in_array('ir:citationCModel', $object->models) &&
+    in_array(('ir:citationCModel' || 'ir:thesisCModel'), $object->models) &&
```

Comment:
Introducing an `OR` like this in PHP doesn't really work, I'm afraid. What I think you may want to do instead is an intersection, so something probably like:

```php
array_intersect(array('ir:citationCModel', 'ir:thesisCModel'), $object->models)
```

If there's something in the intersection, the result will be a non-empty array, and a non-empty array has a truth value of `TRUE` in PHP.

I'm not sure if we want to address it now (or just make a ticket about it), but there seems to be other places where `ir:thesisCModel` wasn't properly introduced, such as the [RoMEO check](https://github.com/Islandora/islandora_scholar/blob/7.x/islandora_scholar.module#L209).